api/server/router/local/image.go

```diff
@@ -292,9 +294,14 @@ func (s *router) postBuild(ctx context.Context, w http.ResponseWriter, r *http.R
 		// Do not write the error in the http output if it's still empty.
```

Comment:
I'm not sure if it matters or not, but I'm wondering why you send the `notVerboseBuffer` output back to the client via the error instead of writing it to `output` and then send the error.